### PR TITLE
Be less verbose on guards and point to the "Guards" page in the docs

### PR DESCRIPTION
**Pull request by whatyouhide**
Makes sense after elixir-lang/elixir#5167 :)

### CODE REVIEW

**Comment by whatyouhide** [comment]
@josevalim ping? :)

**Comment by josevalim** [comment]
I plan to merge this once `defguard` is in so we can tidy it all up at once. :)

**Comment by josevalim** [comment]
@whatyouhide defguard didn't make the cut, so can you please rebase and merge this? Sorry for dropping the ball.

**Comment by whatyouhide** [comment]
@josevalim of course, no problem. Give me 10min :)

**Comment by eksperimental** [comment]
@josevalim didn't make the cut yet?
has it been discarded?

**Comment by whatyouhide** [comment]
@josevalim updated!

**Comment by josevalim** [comment]
Ship it!
-- 


*José Valimwww.plataformatec.com.br
<http://www.plataformatec.com.br/>Founder and Director of R&D*

**Comment by eksperimental** [comment]
let me rephrase myself, because it doesn't sound as I indented
By "didn't make the cut yet?" I mean, that it will not make it? or there's still a change that it will make it.

**Comment by josevalim** [comment]
It won't be in 1.5 so we shouldn't hold this any longer.
-- 


*José Valimwww.plataformatec.com.br
<http://www.plataformatec.com.br/>Founder and Director of R&D*

**Comment by eksperimental** [comment]
thank you!
